tock/projects/models.py

```diff
@@ -185,6 +185,12 @@ def active(self):
     def inactive(self):
         return self.get_queryset().filter(active=False)
 
+    def excluded_from_billability(self):
```

Comment:
What's the distinction between excluded from billability and non-billable? Is it time to add comments to the methods?

Reply:
@amymok I think if this is a really simple clarification to deal with here, let's go ahead, but if it takes any real digging, let's lot a separate issue to better document and clarify it.